sechub-doc/src/docs/asciidoc/documents/shared/configuration/go-gen-supported-source-extensions.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+SOURCE=supported-source-extensions.txt
+
+# This script is intended to generate the go code snippet for constants.go
```

Comment:
Generating the list of allowed extensions is a awesome idea.

Reply:
:-)